pkg/operator/ceph/cluster/osd/topology.go

```diff
@@ -30,3 +41,46 @@ var (
 	// The list of supported failure domains in the CRUSH map, ordered from lowest to highest
 	CRUSHMapLevelsOrdered = append([]string{"host"}, append(CRUSHTopologyLabels, KubernetesTopologyLabels...)...)
 )
+
+// ExtractRookTopologyFromLabels extracts rook topology from labels and returns a map from topology type to value,
+// and a bool indicating if there any invalid labels with a  topology prefix.
```

Comment:
Where is the bool?

Reply:
Didn't update the comment when I updated the func signature. Will fix.